substrate/frame/support/src/traits/hooks.rs

```diff
@@ -227,6 +227,26 @@ pub trait OnRuntimeUpgrade {
 	}
 }
 
+/// See [`Hooks::on_runtime_upgrade`].
```

Comment:
Some better docs here would be nice, and also updating the docs here https://paritytech.github.io/polkadot-sdk/master/frame/traits/trait.Hooks.html#implementation-note-standalone-migrations